internal/controller/helmrelease_controller.go

```diff
@@ -732,7 +732,10 @@ func (r *HelmReleaseReconciler) buildRESTClientGetter(ctx context.Context, obj *
 	var err error
 	switch {
 	case kc.ConfigMapRef != nil && kc.SecretRef == nil:
-		var opts []auth.Option
+		opts := []auth.Option{
+			auth.WithClient(r.Client),
```

Comment:
No need, `authutils.GetRESTConfig()` already does this :+1:

Reply:
That’s true…!